packages/pih-esm-root-config/src/pih-esm-root-config.ts

```diff
@@ -13,4 +13,5 @@ Object.keys(applications).forEach((appName) => {
 promiseBeforeStart.then(start).catch((err) => {
   console.error(`Failed to initialize i18next translations`);
   console.error(err);
+  start();
```

Comment:
This is unrelated -- simply that we should start the application with everything in English if the translation engine fails catastrophically.